src/main.c

```diff
@@ -449,6 +453,22 @@ void print_help(char* programname, struct device* device_found, bool _show_all)
     }
     // ------
 
+    // ------ Category: Bluetooth
+    bool show_bt_when_powered_on = show_all || has_capability(device_found->capabilities, CAP_BT_WHEN_POWERED_ON);
+    bool show_bt_call_volume     = show_all || has_capability(device_found->capabilities, CAP_BT_CALL_VOLUME);
+
+    if (show_rotate_to_mute || show_microphone_mute_led_brightness || show_microphone_volume) {
```

Comment:
Small mistake here: `show_rotate_to_mute` probably left over from copying the code

Reply:
Yessir, fixed

Reply:
Ahh its still there :)

Reply:
Damn didn't see that

Reply:
Sorry :(